transport/websocketpeer_test.go

```diff
@@ -0,0 +1,42 @@
+package transport_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/gammazero/nexus/v3/router"
+	"github.com/gammazero/nexus/v3/transport"
+	"github.com/gammazero/nexus/v3/transport/serialize"
+	"github.com/gammazero/nexus/v3/wamp"
+)
+
+func TestCloseWebsocketPeer(t *testing.T) {
```

Comment:
instead of starting a router and creating a websocket connection you should just mock websocket connection. You test will look something like
```
func TestCloseWebsocketPeer(t *testing.T) {
	logger := log.Default()
	peer := transport.NewWebsocketPeer(newMockSession(), &serialize.JSONSerializer{}, 1, logger, 0, 0)

	// Close the client connection.
	peer.Close()

	// Try closing the client connection again. It should not cause an error.
	peer.Close()
}
```

Reply:
Done